platforms/extensibility/plugin-use/src/main/java/org/gradle/plugin/use/internal/DefaultPluginRequestApplicator.java

```diff
@@ -215,7 +183,7 @@ private static InvalidPluginException exceptionOccurred(PluginRequestInternal re
         return new InvalidPluginException(String.format("An exception occurred applying plugin request %s", request.getDisplayName()), e);
     }
 
-    private static Result resolveToFoundResult(PluginResolver effectivePluginResolver, PluginRequestInternal request) {
+    private static PluginResolution resolveToFoundResult(PluginResolver effectivePluginResolver, PluginRequestInternal request) {
```

Comment:
💅 This method name no longer seems applicable, it's not returning a `Result` any longer.